Documentation/Patterns.md

```diff
@@ -223,7 +224,7 @@ TEST_CASE(string_view_literal_operator)
 ## Source Location
 
 C++20 added std::source_location, which lets you capture the
-callers __FILE__ / __LINE__ / __FUNCTION__ etc as a default
+callers **FILE** / **LINE** / **FUNCTION** etc. as a default
```

Comment:
This is wrong, the variables are named `__FILE__`, `__LINE__` and `__SOURCE__`, it's not formatting. But maybe they should have ` around them.

Reply:
Ok, I'll try to fix these issues, I have noticed that when I saved the files that it automatically changed these things to asterisks. I'll have to look into my own lsp settings for that.